contentScript.js

```diff
@@ -79,7 +79,14 @@ const enterCaptcha = () => {
   }
 
   setTimeout(() => {
-    if (enableautoconfirm) $("ion-button.confirm-btn")[0].click();
+    if (enableautoconfirm) 
+      try{
+        $("ion-button.confirm-btn")[0].click();
```

Comment:
Why would click event throw an exception? How did you test this?

Reply:
I thought that if confirmation doesn't succeed, it would be an exception. Maybe a bad assumption, sorry

Reply:
Don't need to be sorry. That's okay. Try to figure out how you can detect a failure.

Reply:
Couldn't test because I'm unable to reproduce the error easily, as I'd only get to that page if there are any slots available in the first place. I looked at other button click examples and came up with this.

Reply:
yes, will look into ways of detecting confirmation failure

Reply:
If you need available slots, check Gadchiroli district of Maharashtra state - you would get plenty there.

Reply:
no slots available for 18+ unfortunately

Reply:
It looks like a button click that's not successful, for example `Minimum age limit is 45 years for this center`, *does not* throw an exception.